src/block-management/block-holder.js

```diff
@@ -206,8 +229,9 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 			// accessible prop needs to be false to access children
 			// https://facebook.github.io/react-native/docs/accessibility#accessible-ios-android
 			<TouchableWithoutFeedback
-				accessible={ false }
-				accessibilityLabel="block-container"
+				accessible={ ! isSelected }
```

Comment:
I like this, it makes more sense to only make it false when it's selected and we'll only interact with the inner elements when it's selected anyway

Reply:
Yes, it doesn't seem to work on android though...